R/article.R

```diff
@@ -233,6 +233,17 @@ jasa_article <- function(
   )
 }
 
+#' @section \code{josis_article}: Format for creating submissions to the
+#' Journal of Spatial Information Science. Adapted from
+#' \samp{https://www.overleaf.com/latex/templates/journal-of-spatial-information-science-template/bmdhbgxnhtqx}.
+#' @export
+#' @rdname article
+josis_article <- function(..., keep_tex = TRUE, citation_package = 'default') {
```

Comment:
See commend about bibliography in template: I believe it should be natbib here not default